cmd/init.go

```diff
@@ -112,8 +116,20 @@ func (cmd *InitCmd) Run(cobraCmd *cobra.Command, args []string) {
 	// Print DevSpace logo
 	log.PrintLogo()
 
-	// Check if user wants to use devspace cloud
-	cmd.checkIfDevSpaceCloud()
+	cloudRegistryHostname, err := cmd.getCloudRegistryHostname()
```

Comment:
Why is this the first question that is asked? When the user does not want to push an image because for example he selectes the use existing image he doesn't need to specify an image registry